openassessment/fileupload/api.py

```diff
@@ -5,11 +5,19 @@
 
 """
 
-from __future__ import absolute_import
```

Comment:
nit for header comments: Instead of "and provide URLs to the new location" consider: 
"and manage URLs of uploaded files."